edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.Test/DockerModuleTest.cs

```diff
@@ -476,7 +476,7 @@ public void TestSerializeContainerCreateConfig()
         {
             // Arrange
             string createOptions = @"{""Env"": [""k1=v1"",""k2=v2""]}";
-            var config = new DockerConfig("ubuntu:latest", createOptions);
+            var config = new DockerConfig("ubuntu:latest", createOptions, null);
```

Comment:
Besides null, can we add tests to double check none and a real content trust case?

Reply:
Good point. i am right now exploring it